pkg/console/operator/operator.go

```diff
@@ -183,7 +189,7 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 	switch {
 	// v4.0.0 or nil
 	case v311_to_401.BetweenOrEmpty(currentActualVersion):
-		outConfig, configChanged, err = sync_v400(c, outConfig)
+		outConfig, configChanged, err = sync_v400(c, c.recorder, outConfig)
```

Comment:
NIT: So since we added it to the `consoleOperator{ }` struct above, it may not need to be separated out as a separate argument here?  Ie could just be:

```go
// operator.go
// its already part of 'c'
sync_v400(c, outConfig)

// sync_v400.go
func sync_v400(co *consoleOperator, consoleConfig *v1alpha1.Console) {
   // so here we can pop it off to use as you are
   recorder := c.recorder
}
```